src/appliance_setup/avs/avsarconboarder/retriever/vsphere/vsphere_details_retriever.py

```diff
@@ -1,24 +1,81 @@
+import os
+import json
+from urllib.parse import urlparse
 from ...entity._vsphere_resource import VSphereResourceData
 from ...retriever._retriever import Retriever
-
+import logging
+from pkgs._govc_cli import govc_cli
+from pkgs._exceptions import vCenterOperationFailed
 
 class VSphereDetails(Retriever):
     instance = None
     _arc_rp_name = "arc-rp"
     _arc_folder_name = "arc-folder"
-    _data_center = "SDDC-Datacenter"
```

Comment:
Should we stick to older defaults for backward compatibility?

Reply:
added backward compatibility